kv/document_test.go

```diff
@@ -6,16 +6,6 @@ import (
 	influxdbtesting "github.com/influxdata/influxdb/testing"
 )
 
-func TestBoltDocumentStore(t *testing.T) {
```

Comment:
looks like the inmem test for this one is skipped

Reply:
same with the notificaiton rules store oddly enough. No idea what all those are about tbh 🤷‍♂.

Reply:
not a big deal, we will build the inmem transaction in another pr